test("sample test1", () => {
  throw new Error("sample error");
});
